Calendar/SceneDelegate.m

```diff
@@ -15,9 +16,12 @@ @implementation SceneDelegate
 
 
 - (void)scene:(UIScene *)scene willConnectToSession:(UISceneSession *)session options:(UISceneConnectionOptions *)connectionOptions {
-    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
-    UIViewController *homeTabBarController = [storyboard instantiateViewControllerWithIdentifier:@"TabBarController"];
-    self.window.rootViewController = homeTabBarController;
+    PFUser *currentUser = [PFUser currentUser];
+    if (currentUser) {
+        UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
+        UIViewController *homeTabBarController = [storyboard instantiateViewControllerWithIdentifier:@"TabBarController"];
+        self.window.rootViewController = homeTabBarController;
+    }
```

Comment:
It's sometimes nice to have an else with comments. 

```
if (currentUser) {
    UIStoryboard *storyboard = [UIStoryboard storyboardWithName:@"Main" bundle:nil];
    UIViewController *homeTabBarController = [storyboard instantiateViewControllerWithIdentifier:@"TabBarController"];
    self.window.rootViewController = homeTabBarController;
}
else {
    // Is already showing login view on app launch
}
```

I'm not sure how I feel about it. Just thinking it's one of the few valid uses for comments in code imo.